.github/workflows/ci.yml

```diff
@@ -75,7 +81,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: '~1.15.5'
+          go-version: '~1.16.0' # That's the minimum Go version that works with arm.
```

Comment:
Would you mind updating the README accordingly so this gets reflected? That is, that we support `1.16.0` onwards.